CLAUDE.md

```
@@ -164,6 +164,28 @@ Server-side bare git repo via isomorphic-git. Git HTTP smart protocol at `/api/g
+**Env var patterns:**
+- `MASTER_KEY` / `MASTER_KEY_FILE` — single-key legacy pattern, treated as KID `v1`.
+- `MASTER_KEY_<KID>` / `MASTER_KEY_FILE_<KID>` — additional keys, e.g. `MASTER_KEY_v2=<base64>`.
+
+The lexicographically last KID is used for new encryptions; all loaded keys are available for decryption.
+
+**Rotation procedure:**
+1. Generate a new key: `openssl rand -base64 32`
+2. Add it alongside the old key: set `MASTER_KEY_v2=<new-base64>` (keep `MASTER_KEY`/`MASTER_KEY_v1` in place).
+3. Restart the app — new secrets encrypt under `v2`, old secrets still decrypt via `v1`.
+4. Run the migration CLI to re-encrypt existing rows:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)